mud-contracts/common/constants/src/constants.sol

```diff
@@ -15,6 +15,14 @@ bytes14 constant INVENTORY_DEPLOYMENT_NAMESPACE = "eveworld";
 bytes14 constant SMART_DEPLOYABLE_DEPLOYMENT_NAMESPACE = "eveworld";
 bytes14 constant SMART_STORAGE_UNIT_DEPLOYMENT_NAMESPACE = "eveworld";
 
+// placeholder values
+uint256 constant SMART_CHARACTER_CLASS_ID = 123456;
```

Comment:
can we move to the env variables where it takes the Deployable, Character and Inventory typeId's and item Ids? We could get read the values form the .env and hash it to create the classIds by hashing 
eg: `uint256 classId1 = uint256(keccak256(abi.encodePacked("typeId12")));`

Reply:
I have added helper library functions in the ModulesRegistrationLibrary so that we can create SSU or Deployables or Smart Characters by directly using a `typeId` in input and it returns the hashed `classId` directly

It was a bit messy to put this there, agreed